<launch>
 
  <node pkg="pinger_tracker" type="signal_simulator.py" name="signal_simulator" respawn="true">
  </node>

  <node pkg="pinger_tracker" type="phase_shift.py" name="phaseshift" respawn="true">
  </node>

  <node pkg="pinger_tracker" type="pinger_visualizer.py" name="ping_viz" respawn="true">
  </node>  

  <node pkg="rqt_reconfigure" type="rqt_reconfigure" name="rqt_recon" respawn="false">
  </node>

  <node pkg="pinger_tracker" type="crane_method.py" name="crane_method" respawn="true">
  </node>

  <node pkg="pinger_tracker" type="ls_method.py" name="ls_method" respawn="true">
  </node>  

  <node pkg="pinger_tracker" type="bancroft_method.py" name="bancroft_method" respawn="true">
  </node>   

  <node pkg="rviz" type="rviz" name="rviz" respawn="true">
  </node>

</launch>